src/js/state/Columns/touch.test.js

```diff
@@ -0,0 +1,70 @@
+/* @flow */
+import {createColumn} from "./models/column"
+import Columns from "./"
+import actions from "./actions"
+import initTestStore from "../../test/initTestStore"
+import touch from "./touch"
+
+const columns = {
+  "1": [
+    {name: "_path", type: "string"},
+    {name: "duration", type: "interval"}
+  ],
+  "2": [
+    {name: "_path", type: "string"},
+    {name: "ts", type: "time"}
+  ]
+}
+let store
```

Comment:
If you are using a global, doesn't this mean that the tests in this file cannot be run in parallel?

Reply:
Correct, the tests cases in this file are not run in parallel. With Jest, each individual file gets it's own process and global state.

Here's a line from their homepage:

> By ensuring your tests have unique global state, Jest can reliably run tests in parallel.

https://jestjs.io/

Reply:
ok